src/fontations/lib.rs

```diff
@@ -60,6 +61,8 @@ struct FontationsData<'a> {
     char_map: Charmap<'a>,
     outline_glyphs: OutlineGlyphCollection<'a>,
     color_glyphs: ColorGlyphCollection<'a>,
+    glyph_names: GlyphNames<'a>,
+    glyph_from_names: AtomicPtr<HashMap<String, hb_codepoint_t>>,
```

Comment:
The stdlib has a nice synchronization primitive for this called [`OnceLock`](https://doc.rust-lang.org/beta/std/sync/struct.OnceLock.html) so you can replace this type with `OnceLock<HashMap<String, hb_codepoint_t>>` and then the code below becomes much cleaner.